base/bucket.go

```diff
@@ -22,6 +22,7 @@ import (
 	memcached "github.com/couchbase/gomemcached/client"
 	sgbucket "github.com/couchbase/sg-bucket"
 	"github.com/couchbaselabs/walrus"
+	"gopkg.in/couchbase/gocbcore.v2"
```

Comment:
I initially tried to use `github.com/couchbase/gocbcore` but ran into issues, because within gocb it explicitly references `gopkg.in/couchbase/gocbcore...` and it could not find them in the `GOPATH`

I totally agree about using `github.com/couchbase/gocbcore`, but I don't think we can do this without upstream changes from gocb.

@brett19 if you have any insight on this, or if I'm missing something, let me know!  I can provide more details on the issue I ran into if needed.